lib/report_portal/cucumber/report.rb

```diff
@@ -10,39 +14,89 @@ module ReportPortal
   module Cucumber
     # @api private
     class Report
-      def parallel?
-        false
-      end
+      attr_accessor :parallel, :started_launch
 
       def attach_to_launch?
         ReportPortal::Settings.instance.formatter_modes.include?('attach_to_launch')
       end
 
-      def initialize
-        @last_used_time = 0
+      def initialize(logger)
+        @logger = logger
+        ReportPortal.last_used_time = 0
+        ReportPortal.logger = logger
         @root_node = Tree::TreeNode.new('')
         @parent_item_node = @root_node
-        start_launch
+
+        set_parallel_tests_vars
+
+        if ParallelTests.first_process?
```

Comment:
This PR doesn't seem to support the following way to manage reporting to Report Portal

1. A rake task `start_launch` is run to start Report Portal launch.
2. Multiple Cucumber processes are run with `rp_launch_id` environment variable and report results to the same launch
3. When all child Cucumber processes are finished, launch is closed by running `finish_launch` rake task.

The way described above is supported in current master.

Reply:
can you share this rake file so I can make sure it works with it?

Reply:
https://github.com/reportportal/agent-ruby/blob/master/lib/report_portal/tasks.rb

Reply:
As ParallelTests is not a required dependency, it should not be required in runtime if the user doesn't use it.

Reply:
@abotalov can you confirm if this was fixed?

Reply:
they were, you asked to remove them.

Reply:
https://github.com/reportportal/agent-ruby/pull/59/commits/2c447d8affc440c482f773f13b16931385edbfcc
didn't you ask to remove them?

Reply:
I'm not sure what you are asking ...
https://github.com/reportportal/agent-ruby/blob/92af68db52d038bb6c3ddde568ce88757762dcf8/Gemfile#L9,L13

Reply:
I ask to conditionally load dependencies in code only if user requested to run tests in parallel. ReportPortal::Cucumber::Formatter should be usable if user doesn't have parallel_tests nd sys-proctable instealled

Reply:
I don't understand, it's part of the initialization and both libs are required 
https://github.com/reportportal/agent-ruby/blob/efc07c0d0b5cfe8db93a7387a0a8778f45e3e776/lib/report_portal/cucumber/report.rb#L21-L30

what is it that you are worried about? I added 2 dependencies sure users will now need to use a bit more of disk space are we arguing about < MB of space?